### ISSUE TITLE
All functionality of the screen is not operable through a keyboard (Tab) interface Design issue

### ISSUE THREAD

**Issue by ramdev1980** [percussion/percussioncms#801]
**Describe the bug**
All the options (User Created Sites and pages, Action menu, View menu, Publishing menu, Import Summary menu, help menu etc. {need to check others too}) and functionality of the screen is not operable through a keyboard interface (tab) Design issue

**To Reproduce**

1. Go to Design (http://localhost:9992/cm/app/?view=design&site=test&path=%2FSites%2Ftest%2F&pathType=page)
Click on a site.
2. See that all the options/controls on the screen are not accessible using Tab key.
3. Try the above steps for pages under site.

**Expected behavior**
All page elements should be operable through a keyboard interface (tab).

**Additional context**
Navigate all the options available for similar issue and fix.

**Comment by ramdev1980** [comment]
Fixed with https://github.com/percussion/percussioncms/issues/779
